san/batch.py

```diff
@@ -16,7 +17,12 @@ def execute(self):
         batched_queries = []
 
         for idx, q in enumerate(self.queries):
```

Comment:
Let's also rework this to be `query` instead of `q`. The name `q` does not mean anything